fdbclient/NativeAPI.actor.cpp

```diff
@@ -370,10 +370,14 @@ void traceTSSErrors(const char* name, UID tssId, const std::unordered_map<int, u
 	}
 }
 
-ACTOR Future<Void> databaseLogger(DatabaseContext* cx) {
+ACTOR Future<Void> databaseLogger(DatabaseContext* cx, Reference<AsyncVar<ClientDBInfo>> clientDBInfo) {
 	state double lastLogged = 0;
 	loop {
 		wait(delay(CLIENT_KNOBS->SYSTEM_MONITOR_INTERVAL, TaskPriority::FlushTrace));
+		
+		TraceEvent("ClientMetrics", clientDBInfo->get().id)
```

Comment:
Do you know if client to different clusters have different `id` here? trying to differentiate different threads here. @sears